app/views/projects/show.html.erb

```diff
@@ -23,17 +40,78 @@
       $(this).parents('form').trigger('submit.rails');
     });
 
-    $("form").on('ajax:success', function(event, data, status, xhr) {
-      $(this).removeClass('error-saving');
-      $(this).effect("highlight", {color: "#d8f3d8"}, 1000);
+    $(".tagathon-project .content-list form").on(
+      'ajax:success',
+      function(event, data, status, xhr) {
+        $(this).removeClass('error-saving');
+        $(this).effect("highlight", {color: "#d8f3d8"}, 1000);
+      }
+    ).on(
+      'ajax:error',
+      function(event, data, status, xhr) {
+        $(this).addClass('error-saving');
+      }
+    ).each(function(index, el) {
+      ref = $(el).attr('data-ref');
```

Comment:
could you write a comment/move to a separate function so that the intent of this code is a bit clearer.